lib/vsc/administration/vo.py

```diff
@@ -355,12 +386,29 @@ def _set_member_quota(self, storage_name, path, member, quota):
         @type quota: integer (hard value)
         """
         try:
-            hard = quota * 1024 * self.storage[self.host_institute][storage_name].data_replication_factor
-            soft = int(hard * self.vsc.quota_soft_fraction)
+            storage = self.storage[self.host_institute][storage_name]
+        except KeyError:
+            errmsg = "Trying to access non-existent field %s in the storage dictionary"
+            logging.exception(errmsg, storage_name)
+        else:
+            fs_backend, fs_backend_err = storage.load_operator()
+
+        hard = quota * 1024
+        if storage.backend == 'gpfs':
+            hard *= storage.data_replication_factor
+        soft = int(hard * self.vsc.quota_soft_fraction)
 
-            self.gpfs.set_user_quota(soft=soft, user=int(member.account.vsc_id_number), obj=path, hard=hard)
-        except GpfsOperationError:
-            logging.exception("Unable to set USR quota for member %s on path %s", member.account.vsc_id, path)
+        if storage.backend == 'oceanstor':
+            # OceanStor accepts usernames
```

Comment:
should be "only accepts usernames"

Reply:
we moved this part out of vsc-administration in https://github.com/hpcugent/vsc-administration/pull/134/commits/9361a448802de3a723b4df20f00a98f8dd783a97